src/main/java/de/qabel/desktop/inject/config/StaticRuntimeConfiguration.java

```diff
@@ -49,4 +55,45 @@ public Stage getPrimaryStage() {
     public ClientDatabase getConfigDatabase() {
         return configDatabase;
     }
+
+    public void setWindow (Pane window) {
+        this.window = window;
+    }
+
+    public Pane getWindow() {
+        return window;
+    }
+
+    public void loadAboutFiles() {
+        loadThanksFile("/files/thanks_file");
+    }
+
+    private void loadThanksFile(String thanksFilePath) {
+        try {
+            thanksFileContent = readFile(thanksFilePath);
+        } catch (IOException e) {
+            showErrorAlertLoadFiles();
+        } catch (NullPointerException ignored) {
+        }
+    }
+
+    private String readFile(String filePath) throws IOException {
+        try (InputStream thanksFile = System.class.getResourceAsStream(filePath)) {
+            return IOUtils.toString(thanksFile, "UTF-8");
+        } catch (NullPointerException ignored) {
+        }
+        return null;
+    }
+
+    private void showErrorAlertLoadFiles() {
```

Comment:
Take a look at `AbstractController::alert`. This should preferrably be the same code. If you don't extend AbstractController there should be a way to cleanly extract the method to a helper class.